clients/web/src/views/layout/FooterView.js

```diff
@@ -9,9 +9,16 @@ import 'girder/stylesheets/layout/footer.styl';
  * This view shows the footer in the layout.
  */
 var LayoutFooterView = View.extend({
+
+    initialize: function (settings) {
+        var mail = settings.contactEmail; // Has to change
```

Comment:
This should have a fallback value, in case `settings.contactEmail` is not provided.